### PR TITLE
Add ec2-ip

### PR DESCRIPTION
**Pull request by vogre**
Like ec2 but only print the ip without connecting

### CODE REVIEW

File: ec2-ip [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    # Set you bucket name here
+    aws s3 cp --quiet s3://$BUCKET/instances.fzf >(cat)
+}
+
+function choose() {
+    cache="$(get_cache)"
+    chosen=$(echo -e "$cache\n%refresh ec2 cache" | fzf -q "$DEFAULT_FZF_QUERY")
+}
+
+while true; do
```
Comment (arielshaqed) [inline ec2-ip]: Logic might be simpler if the loop condition is based on `$chosen`, e.g. something like (untested, but what could possibly go wrong writing a shell script like this?):
```suggestion
until
    choose
    [[ "$chosen" != "%refresh ec2 cache" ]];  
do
    echo "Refreshing EC2 fuzzy cache..."
    aws lambda invoke --region ${region:-eu-central-1} --function-name ec2-fzf-${region:-eu-central-1}-discover_instances /dev/null >/dev/null
done
if [[ -z "$chosen" ]]; then exit 1;
```

**Review by arielshaqed — APPROVED** [review]
Great, this _is_ useful!

File: ec2-ip (hunk 1 above)
Comment (vogre) [reply ec2-ip]: Only one `fi` was missing 👍

**Review by vogre — COMMENTED** [review]
Thanks!

I've added a TODO comment about deduplicating this script and adding more features, which I'm leaving for future PRs and merging this one as is.

**Pull request merged**